webapp/src/app.tsx

```diff
@@ -2053,7 +2064,9 @@ export class ProjectView
                                 })
                         });
                 } else {
-                    opts.tsOnly = !loadBlocks;
```

Comment:
can we remove/deprecate "tsOnly" entirely? seems like the property is covered by languageRestriction

Reply:
It was only ever used / referenced here and in the ProjectCreationOptions interface, and it's removed from both of those places in this pr

Reply:
oh oops i didn't see you deleted from the interface as well--and importing old projects with tsOnly is fine?

Reply:
Yup, that was never persisted in pxt.json so it will just load like any project without blocks; it only ever did this before:

https://github.com/microsoft/pxt/pull/6499/files#diff-325ec83a257a791d33bd4484fd67fb24L1979